antismash/config/executables.py

```diff
@@ -29,7 +29,13 @@
 }
 
 _NO_KNOWN_ALTS = ["hmmsearch", "hmmpress", "hmmscan", "meme", "fimo", "glimmerhmm",
-                  "prodigal", "muscle", "java", "blastp", "makeblastdb"]
+                  "prodigal", "muscle", "java", "blastp", "makeblastdb", "pplacer", "guppy"]
+
+## NOTE: not sure if this is the right place for this, but this was necessary on my box for pplacer and guppy to run correctly
```

Comment:
This isn't necessary on a Debian system with the usual `LANG` environment variables, this is a bit of a nuclear approach to what is really a per-system setting. 

I don't think it should be included at all, but if absolutely necessary it should be limited in scope to the actual guppy/pplacer executions.